lib/insights_cloud/async/vmaas_reposcan_sync_task.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'vmaas_reposcan_sync_task_base'
+
+module InsightsCloud
+  module Async
+    # Triggers VMaaS reposcan sync when repositories are synced
+    class VmaasReposcanSyncTask < VmaasReposcanSyncTaskBase
+      def plan(repo, *_args)
```

Comment:
Keep *args to stay compatible with Katello::Repository::Sync#plan
see https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/sync.rb#L23